test/jdk/java/lang/StringBuilder/Exceptions.java

```diff
@@ -65,59 +64,68 @@ private static void tryCatch(String s, Throwable ex, Runnable thunk) {
             pass(s);
     }
 
-    public static void main(String [] args) {
+    public static void main(String[] args) {
         System.out.println("StringBuilder()");
         tryCatch("  no args", null, new Runnable() {
-                public void run() {
-                    new StringBuilder();
-                }});
+            public void run() {
+                new StringBuilder();
+            }
+        });
```

Comment:
Nit: it'd be good to make the formatting of all calls to tryCatch method consistent.